### ISSUE TITLE
Babel plugin cannot be found

### ISSUE THREAD

**Issue by balasan** [albertogasparin/react-magnetic-di#6]
The babel plugin 'cannot be found' at the moment if using `react-magnetic-di/babel` in babel config.

Workaround: reference the full babel plugin path:
```
  // ... other stuff like presets
  plugins: [
    // ... other plugins
    'react-magnetic-di/lib/cjs/babel',
  ],
```

**Comment by princed** [comment]
Can confirm the same problem.

**Comment by dpisani** [comment]
Also have the same problem using version 2.0.0

**Comment by albertogasparin** [comment]
Thank you so much for early testing and finding this issue. Got too excited and published a mayor instead of a beta 😄 
Merged #9 and published v2.0.1: `react-magnetic-di/babel-plugin` is the correct location. I've also addressed a couple of other small issues as well.

**Comment by princed** [comment]
Thanks for all the fixes @albertogasparin, works for us now! Great to have some  enjoyable DI! :)
